src/main/java/Duke.java

```diff
@@ -1,10 +1,241 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    /** count tracker for number of tasks in array */
+    private static int taskCount = 0;
+
+    /** count tracker for number of uncompleted tasks in array */
+    private static int numOfUncompletedTasks = 0;
+
+    /** Array of tasks */
+    private static final Task[] tasks = new Task[100];
+
+    public static void greeting() {
+        System.out.println("_____________________________________________________");
```

Comment:
Since you use this ________________ quite a lot, maybe you could introduce it as a global constant